ui/components/MesheryMeshInterface/PatternService/helper.js

```diff
@@ -76,6 +76,70 @@ function getXKubenetesToRJSFCompatibleFieldType(schema) {
   return returnedType || false;
 }
 
+// Order of properties in the form
+const sortOrder = [
+  "string",
+  "integer",
+  "number",
+  "boolean",
+  "array",
+  "object"
+];
+
+/**
+ * Sorts the properties of the jsonSchema in the order of the sortOrder.
+ * @param {*} properties
+ * @param {*} sortOrder
+ * @returns
+ */
+export const sortProperties = (properties) => {
+  const sortedProperties = {};
+  Object.keys(properties)
+    .sort((a, b) => {
+      return (
+        sortOrder.indexOf(properties[a]?.type) - sortOrder.indexOf(properties[b]?.type)
+      );
+    })
+    .forEach(key => {
+      sortedProperties[key] = properties[key];
+      if (properties[key]?.properties) {
+        sortedProperties[key].properties = sortProperties(
+          properties[key].properties,
+          sortOrder
+        );
+      }
+
+      if (properties[key]?.oneOf || properties[key]?.anyOf || properties[key]?.allOf) {
+        const handleReserve = properties[key].oneOf || properties[key].anyOf || properties[key].allOf;
+        handleReserve.forEach((item, index) => {
+          properties[key].type=item.type; // Adding type as schema lacks type for oneOf, anyOf, allOf
```

Comment:
lets refrain from this statement. The schema is still valid without it. modifying this costs us future maintainance in favor of change in RJSF library.

Reply:
Addionally, I don't see any usage of this later anywhere.